src/seedu/addressbook/AddressBook.java

```diff
@@ -874,12 +859,12 @@ private static void deletePersonFromAddressBook(int index)
 	 */
 	private static boolean deletePersonFromAddressBook(HashMap<PersonProperty, String> exactPerson)
 	{
-		final boolean changed = ALL_PERSONS.remove(exactPerson);
-		if(changed)
+		final boolean isChanged = ALL_PERSONS.remove(exactPerson);
```

Comment:
Sometimes hasChanged is a better name for boolean flags that stores state change status. Consider the context.